tests/integration/go_ethereum/common.py

```diff
@@ -62,7 +63,16 @@ def test_eth_estimateGas_with_block(self,
         )
 
     def test_eth_signTransaction(self, web3, unlocked_account):
-        super().test_eth_signTransaction(web3, unlocked_account, GETH_SIGNED_TX)
+        if 'v1.8.22' in web3.clientVersion:
+            super().test_eth_signTransaction(web3, unlocked_account, GETH_18_SIGNED_TX)
+        else:
+            super().test_eth_signTransaction(web3, unlocked_account, GETH_17_SIGNED_TX)
+
+    def test_eth_submitHashrate(self, web3):
+        if 'v1.8.22' in web3.clientVersion:
+            # https://github.com/ethereum/go-ethereum/commit/51db5975cc5fb88db6a0dba1826b534fd4df29d7
+            pytest.xfail('eth_submitHashrate endpoint updated in 1.8.22 to ethash_submitHashRate')
+        super().test_eth_submitHashrate(web3)
```

Comment:
From what I can tell - geth no longer supports the `eth_submitHashrate` endpoint. What's the strategy for `web3.eth.submitHashrate`? Better to just leave it alone and let users running the latest geth run into `"this method is not available"` errors?